src/shared/util/split-events.spec.js

```diff
@@ -1,8 +1,17 @@
 import { splitEvents } from './split-events';
 import { expect } from 'chai';
+import * as MockDate from 'mockdate';
+
+MockDate.set('Thu Jun 23 2016 14:10:56 GMT+0100 (BST)');
```

Comment:
Would it be preferable to set this within the context of the test runner, e.g. `beforeEach`, or does it only apply to the scope this module?